src/Akkling/Actors.fs

```diff
@@ -209,6 +209,13 @@ and [<Struct>]AsyncEffect<'Message>(asyncEffect: Async<Effect<'Message>>) =
                     runAsync effect |> Async.StartAsTask
                 upcast task ))
 
+and [<Struct>]CombinedEffect<'Message> (x: Effect<'Message>, y: Effect<'Message>) =
```

Comment:
Since `CombinedEffect` is always upcasted into `Effect` interface upon creation, I'm not really sure if making in struct will help. But with compilers you can never be sure without profiling. I'll merge it as it is.